Source/WebCore/Modules/mediastream/CanvasCaptureMediaStreamTrack.h

```diff
@@ -60,6 +60,7 @@ class CanvasCaptureMediaStreamTrack final : public MediaStreamTrack {
         void ref() const final { ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Source, WTF::DestructionThread::MainRunLoop>::ref(); }
         void deref() const final { ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Source, WTF::DestructionThread::MainRunLoop>::deref(); }
         ThreadSafeWeakPtrControlBlock& controlBlock() const final { return ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Source, WTF::DestructionThread::MainRunLoop>::controlBlock(); }
+        size_t weakRefCount() const final { return ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Source, WTF::DestructionThread::MainRunLoop>::weakRefCount(); }
```

Comment:
Having to add another virtual method for classes implementing `ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr` seems like a very unfortunate requirement.